sections/collage.liquid

```diff
@@ -56,17 +56,19 @@
                 card_product: block.settings.product,
                 media_aspect_ratio: 'adapt',
                 show_secondary_image: block.settings.second_image,
-                extend_height: true
+                extend_height: true,
+                wrapper_class: section.settings.card_styles
               %}
             {%- when 'collection' -%}
               {% render 'card-collection',
                 card_collection: block.settings.collection,
                 media_aspect_ratio: 'adapt',
                 columns: 2,
-                extend_height: true
+                extend_height: true,
+                wrapper_class: section.settings.card_styles
               %}
             {%- when 'video' -%}
-              <div class="collage-card color-{{ settings.card_color_scheme }}">
+              <div class="collage-card {% if section.settings.card_styles == '' %}global-media-settings{% else %}product-card-wrapper color-{{ settings.card_color_scheme }}{% endif %}">
```

Comment:
Even when we have all card types with the same styles (default), changing card styles results in the background color changing for these video blocks.

https://user-images.githubusercontent.com/37168033/181635403-a65a4e8e-95c3-46e3-8991-06962d8a6f71.mp4

Is this intentional? I think it's because one has `color-{{ settings.card_color_scheme }}` while the other doesn't. In the original implementation, `color-{{ settings.card_color_scheme }}` was always there.

Reply:
If I understand your question correctly. this is expected.

Media doesn't have a color scheme, so it only follows the color scheme once it is selected to follow product cards. Does that make sense? (it follows all settings from prod cards)

https://screenshot.click/29-41-x4t2y-b74jc.mp4